controllers/business.controller.js

```diff
@@ -163,7 +162,7 @@ const updateVotesAura = async (businessId, options) => {
     // UPVOTE
     // Reassign voter.aura array into a temp storage
     // auraArr is the temp storage.
-    let auraArr = [];
```

Comment:
Same problem with Emily yesterday, lets keep this 'let'. code wont work with a 'const'